core/src/eval/misc.rs

```diff
@@ -12,7 +12,9 @@ pub fn codesize(state: &mut Machine) -> Control {
 
 #[inline]
 pub fn codecopy(state: &mut Machine) -> Control {
-	pop_u256!(state, memory_offset, code_offset, len);
+	pop_usize!(state, memory_offset);
+	pop_u256!(state, code_offset);
```

Comment:
I wonder if there is a way to make `code_offset` a `usize` right away as well. Looking at the `copy_large` implementation the value is ultimately coerced down into a `usize` anyway.